framework/src/media/audio/audio_manager.c

```diff
@@ -539,10 +539,24 @@ static audio_manager_result_t get_audio_volume(audio_io_direction_t direct)
 	}
 	
 	if (direct == INPUT) {
+		if (g_actual_audio_in_card_id == INVALID_ID) {
```

Comment:
This code is now included in line 536.
So "line 542 to 548" and "line 553 to 559" need to be removed...

Reply:
You're right. Fixed.